up42/initialization.py

```diff
@@ -54,6 +54,7 @@ def initialize_project(project_id: Optional[str] = None) -> "Project":
     Args:
         project_id: The UP42 project id
     """
+    warn("Projects will be deprecated soon.")
```

Comment:
Here and below:
* I'm not sure about "soon". The functionality is either active or deprecated (meaning = will be removed in the nearest future). So I'd go with "X are deprecated".
* We can reuse the text I will be putting up as warnings on relevant doc pages:
"X are deprecated. The current analytics platform will be discontinued after January 31, 2024, and will be replaced by new processing functionalities." @seedlit

Reply:
I think we have a date right? Q1 2024 according to @kodamaaiko

Reply:
@seedlit Sounds good to me!